app/celery/tasks.py

```diff
@@ -624,3 +624,28 @@ def process_returned_letters_list(notification_references):
             updated, updated_history, len(notification_references)
         )
     )
+
+
+@notify_celery.task(name='send-notify-no-reply')
+@statsd(namespace="tasks")
+def send_notify_no_reply(data):
+    payload = json.loads(data)
+
+    service = dao_fetch_service_by_id(current_app.config['NOTIFY_SERVICE_ID'])
+    template = dao_get_template_by_id(current_app.config['NO_REPLY_TEMPLATE_ID'])
+
+    saved_notification = persist_notification(
```

Comment:
Can we wrap this in a `try..catch` so we make sure the error is logged and in any case, the notification is sent to the queue? Looking at the `persist_notification` function, it seems exceptions will bubble up for now and interrupt the notification to be placed on the queue.